app/Models/Extensions/ThrowsConsistentExceptions.php

```diff
@@ -102,7 +102,7 @@ public function delete(): bool
 	/**
 	 * Serializes this object into an array.
 	 *
-	 * @return array The serialized properties of this object
+	 * @return array<string> The serialized properties of this object
```

Comment:
This is not true. The array representation of a serialized object is not necessarily an array of strings. The keys (aka attribute names) are strings, but the values can be valid JSON types.

```suggestion
	 * @return array<string, string|int|float|null|bool|array> The serialized properties of this object
```

The problem is that this type definition must always stay incomplete, because it is recursive. The second `array` in the definition is not an arbitrary array either, but again an array which maps strings to JSON types.

That is why I decided to just write `array`. Did PhpStan complain?